### PR TITLE
add pt-br/pod (#3181)

### PR DESCRIPTION
**Pull request by juanozorio**
### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/pt-br/pod.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Pod
+status: Completed
+category: conceito
+tags: ["infraestrutura", "fundamento", ""]
+---
+
+Em um ambiente [Kubernetes](/kubernetes/), um pod atua como a unidade de implantação mais básica.
```
Comment (edsoncelio) [inline content/pt-br/pod.md:8]: When the page exists in Portuguese, you must change the link to point to the pt-br version.
Example: `[Kubernetes](/kubernetes/)`-> `[Kubernetes](/pt-br/kubernetes/)`

You can do this to all the links.

File: content/pt-br/pod.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Ele representa um bloco essencial para implantar e gerenciar aplicações containerizadas.
+Cada pod contém uma única instância de aplicação e pode conter um ou mais [containers](/container/).
```
Comment (edsoncelio) [inline content/pt-br/pod.md:10]: containers -> contêineres
Please check all the other occurrences.

**Review by edsoncelio — CHANGES_REQUESTED** [review]
Thanks @juanozorio!
There are just a few changes, please let me know when you finish.

File: content/pt-br/pod.md (hunk 1 above)
Comment (juanozorio) [reply content/pt-br/pod.md]: Done ✅

File: content/pt-br/pod.md (hunk 2 above)
Comment (juanozorio) [reply content/pt-br/pod.md]: Done ✅

**Review by jessicalins — APPROVED** [review]
looks good to me, thanks!

**Review by edsoncelio — APPROVED** [review]
Thank you for your contribution @juanozorio!

**Pull request merged**